drivers/pulseaudio/SCsub

```diff
@@ -2,4 +2,6 @@
 
 Import("env")
 
+if "pulseaudio" in env:
```

Comment:
That would include pulseaudio if you pass `pulseaudio=no` too. You should also check `and env["pulseaudio"]`.

Reply:
Done, thanks!